Makefile

```
@@ -20,7 +21,7 @@ ifeq 'yes' "$(shell test -f $(BINDIR)/kubebuilder && echo -n 'yes')"
 	KUBEBUILDER_ASSETS ?= $(BINDIR)
 endif
 
-all: test build
+all: test skaffold-build
```

There is no target for `skaffold-build`.

```suggestion
all: test build
```